js/Board.js

```diff
@@ -27,5 +27,15 @@ function initSortable() {
 		connectWith: '.card-list',
 		placeholder: 'card-placeholder',
 		opacity: 0.95,
+		receive: function(event, ui) {
+			var movedCard = ui.item[0];
+			$.ajax({
+				url: baseUrl + '/card/' + self.id,
```

Comment:
`self.id` is unresolved here so your request go to `https://kodilla.com/pl/bootcamp-api/card/undefined` :)